pages/pay-grades.md

```diff
@@ -16,7 +16,7 @@ This information is intended as a guideline only. Use this information to help t
 
 |               | **Grade 13**      |**Grade 14**|**Grade 15**|
 | -----------------:|:-------------:| :-----:| :-----:|
-| **Base Compensation**     | $73,115 to $95,048 | $86,399 to  $112,319 |$101,630 to $132,122|
+| **Desired Base Compensation**     | $73,115 to $95,048 | $86,399 to  $112,319 |$101,630 to $132,122|
```

Comment:
@fureigh I see your point and I think it raises some more general confusion that leads me to think that we should pull salary away from these other items. 

The other items in the table describe the level of experience the applicant needs to be qualified for that level. Putting Salary next to them makes me feel like it's saying that those are the salary's that you should have to qualified for that level. I think that's the issue I was having/trying to address by calling out desired there.